includes/dashboard/viewUsers.php

```diff
@@ -0,0 +1,148 @@
+<?php
+
+/* 
+ * this is to view users category wise
+ */
+session_start();
+
+if( !( isset( $_SESSION['Username'] ) && isset($_SESSION['Name']) && $_SESSION['Username'] === 'admin' ) )
+{
+	echo "To access this page you need to login as Admin. Please log in first.";
+	die();
+}
+
+
+require_once "../../LocalSettings.php";
+require_once "../Globals.php";
+
+
+
+
+echo "<p>To vied the users groupwise, select the group and click 'view selected group'</p><br><br>";
+
+
+$sqlConnCat = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $catDB );
+
+if( $sqlConnCat->connect_errno ) 
+{
+	echo "Internal Server Error, Contact Administrator.";
+	die();
+}
+
+
+$sqlConn = new mysqli( $eorgDBserver , $eorgDBuser , $eorgDBpasswd , $eorgDBname );
+
+if( $sqlConn->connect_errno ) 
+{
+	echo "Internal Server Error, Contact Administrator.";
+	die();
+}
+
+
+$qryCat = "SELECT TABLE_NAME FROM information_schema.tables WHERE table_schema = ?";
+$stmtCat = $sqlConnCat->prepare( $qryCat );
+$stmtCat->bind_param ( 's', $catDB );
+
+
+if ( ! $stmtCat->execute() )
+{
+	echo"there is a problem with database<br>";
+	die ();
+}
+
+$resCat = $stmtCat->get_result();
+
+if ( $resCat->num_rows == 0 )
+{
+	echo "Sorry, first set the broad categories<br>";
+	die ();
+}
+
+while ($rowCat = mysqli_fetch_row ($resCat))
+{
+	$newRowCat = str_replace('_', ' ', $rowCat[0]);
+	$qry2Cat = "SELECT * FROM $rowCat[0] LIMIT 1"; // to take the level no. of the broad category
+	$stmt2Cat = $sqlConnCat->prepare ( $qry2Cat );
+	if ( ! $stmt2Cat->execute () )
+	{
+		echo "Sorry problem with database<br>";
+		die ();
+	}
+	$res2Cat = $stmt2Cat->get_result ();
+	$row2Cat = mysqli_fetch_row ( $res2Cat );
+	$valueToPass = $rowCat[0].",".$row2Cat[0] ;
+	echo "<input type='radio' name='user' value=$valueToPass onclick='startAskingOptions(this.value)'>$newRowCat";
+	$stmt2Cat->close();
+}
+
+$stmtCat->close();
+
+
+$html = <<<HTML
+<div id='askUserOptions'>
+</div><div id='backGround'>
+</div>
+<button type='button' onclick="showSelectedGroup(document.getElementById('backGround').innerHTML)">View selected group</button>
+<div id='users'>
+HTML;
+
+echo $html;
+
+
+echo "<h2><br><br><br>The available users are :<br><br></h2>";
+
+
+
+
+$i = 1;
+
+$html = <<<HTML
+<table>
+
+<tr>
+<th>Sr. No.</th>
+<th>User ID</th>
+<th>Name</th>
+<th>Sex</th>
+<th>Designation(s)</th>
+</tr>
+HTML;
+
+echo $html;
+
+
+$qry = "SELECT * FROM $loginDB " ;
+$stmt = $sqlConn->prepare ( $qry );
```

Comment:
__[code_sniffer]__
* Space before opening parenthesis of function call prohibited
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->